### PR TITLE
Remove add_import script as support for package imports has been added upstream.

### PR DESCRIPTION
**Pull request by Skarlso**
### Description

Closes https://github.com/weaveworks/eksctl/issues/4994

Support for adding a dot import has been added in ifacemaker. We pin to that specific version in the requirements.txt.

### Checklist
- [ ] Added tests that cover your change (if possible)
- [ ] Added/modified documentation as required (such as the `README.md`, or the `userdocs` directory)
- [x] Manually tested
- [x] Made sure the title of the PR is a good description that can go into the release notes
- [x] (Core team) Added labels for change area (e.g. `area/nodegroup`) and kind (e.g. `kind/improvement`)

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

**Review by aclevername — APPROVED** [review]

**Comment by Skarlso** [comment]
OH I have to update the build image eh?

**Comment by Skarlso** [comment]
Ok, pushed the images.

File: build/docker/build_image_manifest [hunk 1]
```diff
@@ -13,7 +13,7 @@
 "k8s.io/code-generator v0.22.1"
 "k8s.io/code-generator v0.22.1"
 "sigs.k8s.io/mdtoc v1.1.0"
-"github.com/vburenin/ifacemaker v1.1.0"
+"github.com/vburenin/ifacemaker v1.2.0"
```
Comment (Skarlso) [inline build/docker/build_image_manifest:16]: This is crap. It copies the mod import but not the replace?

Comment (Skarlso) [reply build/docker/build_image_manifest:16]: I'll update the imported version then. _sigh_

**Review by cPu1 — APPROVED** [review]

**Review by nikimanoledaki — APPROVED** [review]

**Pull request merged**